core/src/Cabana_LinkedCellList.hpp

```diff
@@ -75,21 +76,33 @@ class LinkedCellList
       \param grid_min Grid minimum value in each direction.
 
       \param grid_max Grid maximum value in each direction.
+
+      \param neighborhood_radius Radius for neighbors
+
+      \param cell_size_ratio Ratio of the cell size to the neighborhood size
     */
     template <class SliceType>
     LinkedCellList(
         SliceType positions, const typename SliceType::value_type grid_delta[3],
         const typename SliceType::value_type grid_min[3],
         const typename SliceType::value_type grid_max[3],
+        const Scalar neighborhood_radius, const Scalar cell_size_ratio = 1,
```

Comment:
We require `const typename SliceType::value_type` for all the other scalars, so we can do the same for the cutoff

Reply:
Tried this, but to have the stencil as a member, we need a Scalar type to be used for it and I couldn't get it to work. I'm sure it's possible but I'm not sure how to setup the templating correctly for it